docusaurus/src/remark/docsHeaderDecoration.js

```diff
@@ -0,0 +1,135 @@
+const fetch = require("node-fetch");
+const visit = require("unist-util-visit");
+
+const registry_url =
+  "https://connectors.airbyte.com/files/generated_reports/connector_registry_report.json";
+let registry = {};
+let loading = false;
+
+const plugin = () => {
+  const transformer = async (ast, vfile) => {
+    if (!isDocsPage(vfile)) return;
+
+    await fetchCatalog();
+
+    const pathParts = vfile.path.split("/");
+    const connectorName = pathParts.pop().split(".")[0];
+    const connectorType = pathParts.pop();
+    const dockerRepository = `airbyte/${connectorType.replace(
+      /s$/,
+      ""
+    )}-${connectorName}`;
+    const registryEntry = registry.find(
+      (r) => r.dockerRepository_oss === dockerRepository
+    );
+
+    if (!registryEntry) return;
+
+    visit(ast, "heading", (node) => {
+      let headerFound = false;
+      if (node.depth === 1 && node.children.length === 1 && !headerFound) {
+        headerFound = true;
+        const originalTitle = node.children[0].value;
+        const originalId = node.data.hProperties.id;
+
+        node.type = "html";
+        node.children = undefined;
+        node.value = buildConnectorHTMLContent(
+          registryEntry,
+          originalTitle,
+          originalId
+        );
+      }
+    });
+  };
+  return transformer;
+};
+
+const buildConnectorHTMLContent = (
+  registryEntry,
+  originalTitle,
+  originalId
+) => {
+  const htmlContent = `
+  <h1 id="${originalId}">${originalTitle}</h1>
+  <small>
+    <table>
+      <tbody>
+        <tr>
+          <td rowSpan="4">
+            <img style="max-height: 75px" src="${registryEntry.iconUrl_oss}" />
```

Comment:
Design suggestion: maybe it would look nicer if we just put the icon in front of the title and just have this table contain the facts. It would also render a bit better on mobile, since we only have one column.